apps/documents/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.urls import path
+
+from apps.documents import views
+
+app_name = "documents"
+
+urlpatterns = [
+    path("<slug:tab_name>", views.RepositoryHome.as_view(), name="repositories"),
+    # Files
+    path("files/", views.FileListView.as_view(), name="files_list"),
+    path("files/<int:id>/details", views.FileDetails.as_view(), name="file_details"),
+    path("files/upload", views.upload_files, name="upload_files"),
+    path("files/<int:id>/delete", views.delete_file, name="delete_file"),
+    path("files/<int:id>/edit", views.edit_file, name="edit_file"),
+    # Collections
+    path("collections/", views.CollectionListView.as_view(), name="collections_list"),
```

Comment:
Should we rename the model to `Collection`? I think it would be good to start off with having the same terminology.

Reply:
Are you suggesting we then rename it again to `Repository` when we introduce RAG, or introduce the notion of collection types then?

Reply:
I was assuming we'd use the same name. It's not exactly clear to me how we'll incorporate the RAG collections in the UI i.e. whether they should be entirely separate or kept together and differentiated with filtering and UI hints e.g. icons

Reply:
Might be good to give Brian a demo and get some input